src/requirements/checkers.ts

```diff
@@ -104,6 +106,22 @@ export const courseIsFWS = (course: Course): boolean =>
 export const courseIsForeignLang = (course: Course): boolean =>
   FLcourses.some(language => course.subject?.includes(language) ?? false);
 
+/**
+ * Call this function to check whether distribution is included in the course
+ * or if the course is on the Other Yes List.
+ *
+ * @param course course object with useful information retrived from Cornell courses API.
+ * @returns if the distribution is included
+ */
+export const hasCategory = (course: Course, distribution: string) => {
+  const otherYesCourse = otherYesCourses.find((x: CourseWithId) => x.crseId === course.crseId);
+  // console.log(otherYesCourse?.categories?.includes(distribution));
+  return (
+    course.catalogDistr?.includes(distribution) ||
+    otherYesCourse?.categories?.includes(distribution)
```

Comment:
Is the optional chaining after `categories` necessary--will the property `categories` every be absent on `otherYesCourse`, if it exists?